AngularApp/projects/app-service-diagnostics/src/armResourceConfig/microsoft.signalrservice/signalr/config.json

```
@@ -9,42 +9,40 @@
     "searchSuffix": "signalr",
     "azureServiceName": "Azure SignalR Services",
     "armApiConfig": {
-		"armApiVersion": "2020-05-01"
-	},
+        "armApiVersion": "2020-05-01"
+    },
     "isSearchEnabled": true,
-	"keystoneDetectorId": "",
+    "keystoneDetectorId": "",
     "liveChatConfig": {
         "isApplicableForLiveChat": false,
         "supportTopicIds": []
     },
     "categories": [
         {
-            "id": "OverallCheck",
-            "name": "Overall Check",
-            "description": "An overall check run to help quickly identify some possible common issues. ",
-            "keywords": [ "Overall Check" ],
-            "color": "rgb(186, 211, 245)",
-            "createFlowForCategory": false,
-            "chatEnabled": false
-        },
-        {
-            "id": "AvailabilityAndPerformance",
+            "id": "AvailabilityAndPerformanceSignalR",
             "name": "Availability And Performance",
             "description": "Investigate issues with your azure signalr service. Find out the common causes and investigate in detail. ",

```

Can we reduce the word count for the description? 
```suggestion
            "description": "Investigate common issues impacting the health of your SignalR Service.",

```

---

Hi @ruimhu, is it possible to use the description I suggested? I tested the new descriptions you've added, but they don't take up the same number of lines, so the cards don't render as nicely.